### DISCUSSION TITLE
Multiple output items in the same combobox, possible?

### DISCUSSION THREAD

**Discussion by gavinhewitt** [htmlstreamofficial/preline#386] [Q&A] [answered]
Hello!

I'm very very new to preline. Awesome stuff! However, for this job there are a few different designs for the combobox items and I'm wondering how to achieve that, or if it's even possible. Is it?

**Comment by jahaganiev** [comment, marked as answer]
Hey @gavinhewitt - you may check out this demo for multiple outputs https://preline.co/docs/searchbox.html#json-examples-with-tab-filter

It's sorted to different categories with tabs.

Hope that helps!

**Reply by gavinhewitt** [reply]
Wow, super quick reply! Thanks, that should help :)

(missed that, sorry!)
